api/src/test/java/com/graphhopper/util/DistanceCalc3D_DiffblueTest.java

```diff
@@ -0,0 +1,49 @@
+/**
+ *
+ *
+ * Copyright 2018 Diffblue Limited
+ *
+ * Diffblue Limited licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.graphhopper.util;
+
+import com.graphhopper.util.DistanceCalc3D;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class DistanceCalc3D_DiffblueTest {
```

Comment:
@peterschrammel Because we'd need to append a license header with our name in it to the files (sometimes *Test.java already exists - and I don't want to overwrite the license header that's already in there)